src/main/java/nextstep/subway/WebControllerAdvice.java

```diff
@@ -0,0 +1,28 @@
+package nextstep.subway;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.bind.annotation.RestControllerAdvice;
+
+import nextstep.subway.exception.BadRequestException;
+import nextstep.subway.exception.NotFoundException;
+
+@RestControllerAdvice
+public class WebControllerAdvice {
+    private final Logger log = LoggerFactory.getLogger(this.getClass());
```

Comment:
이왕 이렇게 가시는거 정적 상수로 선언하시죠 😄 
```suggestion
    private static final Logger LOG = LoggerFactory.getLogger(this.getClass());
```

Reply:
정적 상수로 가게되면 this.getClass() 사용이 불가능해져서 저는 인스턴스 변수로 사용하는 편입니다. Static으로하면 클래스를 명시해야하는데 복붙하다가 엉뚱한 클래스가 딸려들어가는 경우가 있더라고요. 아니면 차라리 롬복을 쓰는게 나으려나요?^^; 의견 부탁드립니다.

Reply:
네 맞습니다 저도 복붙하다가 this가 아닌 엉뚱한 클래스를 😅 
아닙니다 그렇다고 롬복까지 가실필요는 없어보이십니다 😄 
지금도 충분합니다 🙇

Reply:
머지후에도 피드백 감사드립니다 😄